src/components/BottomSheet/BottomSheet.js

```diff
@@ -39,14 +39,16 @@ const styles = StyleSheet.create({
   container: {
     paddingTop: dimen.normalizeDimen(10),
     paddingHorizontal: dimen.normalizeDimen(20),
-    paddingBottom: dimen.normalizeDimen(20)
+    paddingBottom: dimen.normalizeDimen(20),
+    backgroundColor: COLORS.almostBlack
   },
   containerSheet: (pullBottom) => ({
     borderTopRightRadius: 20,
     borderTopLeftRadius: 20,
-    justifyContent: pullBottom ? 'flex-end' : 'flex-start'
+    justifyContent: pullBottom ? 'flex-end' : 'flex-start',
+    backgroundColor: COLORS.almostBlack
   }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `containerSheet` style are also beneficial. The addition of a background color here ensures consistency across different screens. The use of the `pullBottom` prop to adjust the `justifyContent` property is a clever way to handle different layout requirements based on whether the bottom sheet should pull from the bottom or not. Again, please ensure that `COLORS.almostBlack` is accessible and provides sufficient contrast.



<!-- This is an auto-generated comment by OSS CodeRabbit -->